src/main/java/frc/robot/subsystems/led/Led2023.java

```diff
@@ -731,13 +731,13 @@ public void set(COLORS_467 color, int preSet) {
 
       if (preSet == 1) {
         start = topStart;
-        end = topEndandMidStart + 1;
+        end = topEndandMidStart;
       } else if (preSet == 2) {
         start = topEndandMidStart;
-        end = midEndandBottomStart;
+        end = midEndandBottomStart-1;
```

Comment:
Why is this one -1, but the rest aren't?

Reply:
to match the # of leds that show when 1/3 is set (3 leds consistently)